data/ch/ACCESSCONTROL-ACTIONS-TEST/actions-test.json

```diff
@@ -12822,22 +12822,6 @@
       "leftIcon": "editor:insert-chart",
       "path": "Finance.Set-up.Budget Control Type.Create/Update-Budget Control Type"
     },
-    {
-      "id": 1766,
```

Comment:
Why is this removed??

Reply:
that is removed because that id belongs to RP_APPROVER

Reply:
This action is the reason inbox shows up for all users. 
This is not just used by RP_APPROVER but by most of the employees.

Reply:
ok rolling back